### PR TITLE
Update block arrival histogram values

### PR DESCRIPTION
**Pull request by terencechain**
**What does this PR do? Why is it needed?**
Update the current block arrival histogram bucket value to align with our expectations. The best case is for the block to arrive within 100ms, then the block that arrives between 100 - 500ms is likely the second best. Any block to arrives passed one slot will be considered too late and will likely to be orphaned anyway 

**Which issues(s) does this PR fix?**
N/A

**Other notes for review**
The effort to improve upon the block arrival histogram bucket is we want to begin monitoring block arrival time for reorg against late block proposal. The current bucket is not being looked at much

### CODE REVIEW

File: beacon-chain/sync/metrics.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ var (
 		prometheus.HistogramOpts{
 			Name:    "block_arrival_latency_milliseconds",
 			Help:    "Captures blocks propagation time. Blocks arrival in milliseconds distribution",
-			Buckets: []float64{250, 500, 1000, 1500, 2000, 4000, 8000, 16000},
+			Buckets: []float64{100, 500, 1000, 4000, 8000, 12000},
```
Comment (prestonvanloon) [inline beacon-chain/sync/metrics.go:89]: I think you need even more granularity around 500 to 1000ms. Here's a current screenshot from mainnet metrics 

Screenshot from 2022-09-12 09-17-20

Comment (terencechain) [reply beacon-chain/sync/metrics.go]: Thanks for the chart! I added 250 and 750 back

Comment (prestonvanloon) [reply beacon-chain/sync/metrics.go]: OK great. Are you sure you want to remove 1500 and 2000? That seems like common latency buckets

Comment (terencechain) [reply beacon-chain/sync/metrics.go]: No, you are right. I was worried that too many buckets might be bad, but then I realized there's not much downside to it. Thanks for the suggestions!

File: beacon-chain/sync/metrics.go [hunk 2]
```diff
@@ -86,7 +86,7 @@ var (
 		prometheus.HistogramOpts{
 			Name:    "block_arrival_latency_milliseconds",
 			Help:    "Captures blocks propagation time. Blocks arrival in milliseconds distribution",
-			Buckets: []float64{250, 500, 1000, 1500, 2000, 4000, 8000, 16000},
+			Buckets: []float64{100, 250, 500, 750, 1000, 4000, 8000, 12000},
```
Comment (prestonvanloon) [inline beacon-chain/sync/metrics.go:89]: Please add back 1500 and 2000.

**Review by prestonvanloon — CHANGES_REQUESTED** [review]

File: beacon-chain/sync/metrics.go (hunk 2 above)
Comment (terencechain) [reply beacon-chain/sync/metrics.go]: Ops. I had added it but forgot to push it. Done now

**Review by prestonvanloon — APPROVED** [review]

**Pull request merged**